pages/common/git-fame.md

```diff
@@ -0,0 +1,28 @@
+# git fame
+
+> Pretty-print Git repository contributions.
+> More information: <https://github.com/casperdcl/git-fame>.
+
+- Calculate contributions for the current Git repository:
+
+`git fame`
+
+- Exclude files/directories that match the specified regular expression:
+
+`git fame --excl "{{regular_expression}}"`
+
+- Calculate contributions made after the specified date:
+
+`git fame --since "{{3 weeks ago|2021-05-01}}"`
+
+- Display contributions in the specified format:
+
+`git fame --format {{pipe|md|markdown|yaml|yml|json|csv|tsv|tabulate}}`
```

Comment:
See the comment above

Reply:
I don't think a tldr page makes sense if you have to look up the documentation to get the varous formats. If they all fit in one line, they can be quite valuable.

Reply:
Yes, makes sense.

Reply:
Resolve.